currency/currency.go

```diff
@@ -87,7 +87,7 @@ func ToMinorUnit(currencyCode string, amount float64) (minorUnitAmount uint64, e
 	switch {
 	case amount > 0:
 		factor := getCurrencyFactor(currencyCode)
-		minorUnitAmount = uint64(math.Round(amount * float64(factor)))
+		minorUnitAmount = uint64(math.Round(amount * factor))
 	case amount == 0:
 		minorUnitAmount = 0
 	default:
```

Comment:
do we also remove the `err` here same as `FromMinorUnit`?

Reply:
We cannot as `float64` can be <= 0